receptor-python-worker/pyproject.toml

```diff
@@ -1,3 +1,20 @@
 [build-system]
 requires = ["setuptools"]
 build-backend = "setuptools.build_meta"
+
+[project]
+authors = [
+    {email = "info@ansible.com"},
+]
+description = "The receptor-python-worker command is called by Receptor to supervise the operation of a Python worker plugin."
+dynamic = [
+    "dependencies",
+    "scripts",
+    "version"
+]
+name = "receptor-python-worker"
+
+[project.optional-dependencies]
```

Comment:
By the way, this is another thing that diverges from the metadata in `setup.cfg`. It'll probably be hard to keep both in sync and make sure that `setuptools` behaves correctly across its versions.